tests/conftest.py

```diff
@@ -1090,262 +1085,6 @@ def titanic_v013_multi_datasource_multi_execution_engine_data_context_with_check
     return context
 
 
-@pytest.fixture
-def titanic_pandas_data_context_with_v013_datasource_stats_enabled_with_checkpoints_v1_with_templates(  # noqa: E501
```

Comment:
We don't care for templating with new checkpoints so this whole fixture is outdated